package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "selenium-standalone",
-  "version": "8.3.0",
+  "version": "9.0.0",
```

Comment:
We should revert this as this project has its own pipeline for releasing and bumping the version.

Reply:
to the 8.3.0 ?

Reply:
yes

Reply:
done :)